features/step_definitions/brief_response_steps.rb

```diff
@@ -19,12 +19,15 @@
 end
 
 Given /^I am logged in as the buyer of a closed brief with responses$/ do
-  submitted_brief_response = get_brief_responses('digital-outcomes-and-specialists-3', 'submitted', 'closed').sample
-  @brief = get_brief(submitted_brief_response['brief']['id'])
-  puts "brief id: #{@brief['id']}"
+  submitted_brief_responses = iter_brief_responses('digital-outcomes-and-specialists-3', 'submitted', 'closed')
```

Comment:
Also, do they have to be DOS3 brief responses? Could we look at DOS/DOS2 to bump up the numbers?

Reply:
I don't see why not 🤔

Reply:
We actually have reasonable numbers of closed briefs though, about 150. I don't really understand why it's failing consistently.

Reply:
I think what is happening is that because `get_brief_responses` gets a page at a time and then filters, we've been getting into a situation where it just happens that there is one page which has only one brief response to a closed brief, and that brief just happens to have no active users. It's quite an unlikely circumstance, but because of the pagination we reach a point where we get stuck on it.

That's my theory anyway.

Reply:
Yeah that does kinda make sense